Makefile

```
@@ -0,0 +1,21 @@
+V ?= 0
+
+ifeq ($(V),1)
+Q := 
+ECHO := @true
+else
+Q := @
+ECHO := @echo
+endif
+
+.PHONY: libzr examples clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)